### DISCUSSION TITLE
Changes to css file are not reloaded in docker production

### DISCUSSION THREAD

**Discussion by wilinger** [django-compressor/django-compressor#1109] [General]
django 4.0.3
django-compressor 3.1

I made a change to the content of my main css file but django-compressor is either not recompressing the file or not reloading the new cache  when I restart the container.  When I try to rename the css file, I get  the CommandError below.

entrypoint:
```
      python manage.py collectstatic --noinput
      python manage.py compress --force
      python manage.py collectstatic --noinput
```

template:
```
    {% compress css %}
    <link rel="stylesheet" href='{% static "css/main01.css" %}'>
    {% endcompress %}
```

settings.py:
```
STATICFILES_FINDERS = (
    "django.contrib.staticfiles.finders.FileSystemFinder",
    "django.contrib.staticfiles.finders.AppDirectoriesFinder",
    # other finders
    "compressor.finders.CompressorFinder",
)
STATIC_URL = "/static/"
STATIC_ROOT = BASE_DIR / "staticfiles"
STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
COMPRESS_URL = STATIC_URL
COMPRESS_ROOT = STATIC_ROOT
COMPRESS_STORAGE = "compressor.storage.GzipCompressorFileStorage"
```

When I try to rename main01.css to main02.css I get CommandError
```
CommandError: An error occurred during rendering contact/index.html: 'css/main02.css' could not be found in the COMPRESS_ROOT '/app/staticfiles' or with staticfiles.
```

**Comment by carltongibson** [comment]
Hi. Sorry for your difficulties. This isn't really a compressor issue — there's a billion things that could be going on, not least of which is your container build. (First thought: did `main02.css` actually get included.) 

I'll convert this to a discussion, to see if other people might be able to input, but it's not addressable here. (Generally speaking `collectstatic` is mature and battle-hardened, so it has to be your local/container setup.) Good luck!

**Comment by wilinger** [comment]
Thanks for moving. I mounted the wrong static folder to a docker volume. It's fixed now. Think this can be deleted as a discussion since not really related as well.
